src/components/BooksList.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { removeBook } from '../actions';
+import Book from './Book';
+import '../App.css';
+
+
+const mapStateToProps = (state) => ({
```

Comment:
No need to enclose `state` in `()` when using implicit return :)

Reply:
This couldn't be resolved!
The parentheses are actually required :)

Reply:
Probably i failed to specify what I meant. I meant the parentheses around `state` :D

Reply:
Yeah, I got that but enlist still complains when they are removed. perhaps because the method returns an object?

Reply:
No, i think it's because of something with the ESLint configuration. Anyway haha let's not belabour this too much :D It's ok to keep the parentheses